src/peerd/runtime.rs

```diff
@@ -360,6 +385,52 @@ impl Runtime {
         }
         Ok(())
     }
+    fn reconnect_peer(
+        &mut self,
+        senders: &mut esb::SenderList<ServiceBus, ServiceId>,
+    ) -> Result<(), Error> {
+        // If this is the listener-forked peerd, terminate it.
+        if !self.connect {
```

Comment:
move this outside the function.

Reply:
Done in 0e97200d18ea8f28d52c5c79897947b4605a8f89